packages/documentation/copy/en/reference/ESM Support for Node.md

```diff
@@ -207,7 +207,7 @@ A package author could override this by specifying a separate field called `"typ
 
 The new support works similarly with [import conditions](https://nodejs.org/api/packages.html).
 By default, TypeScript overlays the same rules with import conditions - if you write an `import` from an ES module, it will look up the `import` field, and from a CommonJS module, it will look at the `require` field.
-If it finds them, it will look for a colocated declaration file.
+If it finds them, it will look for a collocated declaration file.
```

Comment:
```suggestion
If it finds them, it will look for a colocated declaration file.
```

I'm not totally sure this one is correct, actually: https://www.merriam-webster.com/dictionary/colocate

Reply:
https://www.merriam-webster.com/dictionary/collocate

Reply:
I spent way too much time trying to compare `colocated`, `collocated` and `co-located`, and it was kinda a tossup.

But, as I write this, I now see: 
![image](https://github.com/microsoft/TypeScript-Website/assets/5341706/2e2943ce-d15e-437b-8625-d6761ca5b90a)


So, who even knows. Probably the hyphenated one is the least weird?

Reply:
Yeah, no worry either way. I usually just defer to the cSpell dictionary unless I know it's one of those odd-balls